wallet/src/main/java/com/android/identity_credential/wallet/PresentationActivity.kt

```diff
@@ -83,22 +70,29 @@ class PresentationActivity : ComponentActivity() {
             deviceEngagement = null
         }
 
-        fun startPresentation(context: Context, transport: DataTransport, handover: ByteArray,
-                              eDeviceKey: EcPrivateKey, deviceEngagement:
-                              ByteArray) {
+        fun startPresentation(
+            context: Context, transport: DataTransport, handover: ByteArray,
+            eDeviceKey: EcPrivateKey, deviceEngagement:
+            ByteArray
+        ) {
```

Comment:
Formatting seems a bit off here. I generally don't comment on formatting (ideally we should automatically reformat source code before it's merged) like that but with the type for `deviceEngagement` sitting on its own line it just looks plain weird and like an oversight!